rpcclient/roundrobin.go

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) 2015-2018 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package rpcclient
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/btcsuite/websocket"
+)
+
+var (
+	// ErrNoConnAvailable indicates no Conn is available for pick().
+	ErrNoConnAvailable = errors.New("no Connection is available")
+
+	// ErrNoUsableConnAvailable indicates no usable connections available i.e.
+	// client is marked as Shutdown.
+	ErrNoUsableConnAvailable = errors.New("no Connection is available for use, all are marked as Shutdown")
```

Comment:
Same here for but the struct name and the connection state.  Also, it's not a grammatically correct error since it contains a comma splice.

I would suggest either: 

```Go
ErrNoUsableConnAvailable = errors.New("no connections are available for use")
```
or 

```Go
ErrNoUsableConnAvailable = errors.New("all connections have been shutdown")
```